newIDE/app/src/MainFrame/index.js

```diff
@@ -895,6 +883,7 @@ class MainFrame extends React.Component<Props, State> {
       previewFirstSceneName,
       isPreviewOverride,
     } = this.state;
+    const layoutOverride = currentProject.getLayout(previewFirstSceneName);
```

Comment:
Seem to me more risky, the value can be not define if previewFirstSceneName isn't yet set.

Reply:
Yes but if you're calling getLayout on a layout that does not exist, you're potentially crashing libGD.js.

Let's avoid this and use `hasLayout` when launching the preview.